src/Task/Base/ParallelExec.php

```diff
@@ -61,11 +66,16 @@ public function getPrinted()
     /**
      * @param bool $isPrinted
      *
+     * @param bool $waitTofinish (optional) Set to false to print messages from
+     *   each process as they are received, instead of batching them for the
+     *   end.
+     *
      * @return $this
      */
-    public function printed($isPrinted = true)
+    public function printed($isPrinted = true, $waitTofinish = true)
```

Comment:
This fluid API is a little awkward:
```
   ->printed(true, false)
```
Maybe this would be better as two (or more) separate APIs?